### DISCUSSION TITLE
Compare builds from a consumed pipeline artifact

### DISCUSSION THREAD

**Discussion by mathieu-meon-eu** [rfennell/AzurePipelines#1935] [Q&A] [answered]
Hi,

Could it be possible to override the "source" and "compared" builds with ones from another build definition ?

I run this task from a "deployment pipeline" which consumes artifacts produced by a "build pipeline" (all yaml multi-stage).

What I would need is not the work item diff between the source versions of the "deployment pipeline" but rather the diff between the runs involved in the consumed "build-pipeline".

I already tried to use "overrideBuildReleaseId" (and "overrideActiveBuildReleaseId") but as it refers to another pipeline it fails with 
_"Unable to locate the current build with id xxxxx in the project xxxxx"._

Thanks a lot.

**Comment by rfennell** [comment, marked as answer]
Sorry know, you can only compare runs of the same pipeline definition. This is because of the underlying Microsoft API calls I make to do the comparison
